client/css/analysis.css

```diff
@@ -0,0 +1,43 @@
+:root {
+    /* the higher the number, the less filled */
+  --percent: 8;
```

Comment:
> the higher the number, the less filled 

Why did you do it like that? Also, why did you put the variable on `:root` instead of on the donut chart container?

You could make the `--percent` variable work like an actual percentage by doing this:
- set the `pathLength` attribute of the `circle` element to 100; then the browser will always do stroke calculations as if the circle's circumference is 100 units (which means a stroke dash offset of 1 will actually offset the stroke by 1%)
- use `calc()` to do subtraction at the site where you use the variable:
  ```css
  stroke-dashoffset: calc(100 - var(--percent));
  ```

Reply:
1. I put the variable on root because there is no container in this pr to add it to.
2. I did it like this so that in JS, I could easily change the `--percent` variable for each donut from there and it would fill the circle accordingly. 440 is approximately 2πr where r is 70 as seen in the code.
   - For example, if Perspective returns 70% toxicity, do 440 - 440*.70 in js and then set the `--percent` variable from there. Very simple. 
3. I tried to set the `pathLength` attribute to 100 and try it that way, but changing the `stroke-dashoffset` had no effect.

Reply:
Check this out: https://jsfiddle.net/jc5pn9rm/

Reply:
And you can use `calc()` if you want the donut to fill up counterclockwise instead of clockwise

Reply:
Very very nice. Thank you